samples/listAssets.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: List Assets
+//   description: List assets under the current project.
+//   usage: node listAssets <ASSET_TYPES>
+//   example: node listAssets "storage.googleapis.com/Bucket,bigquery.googleapis.com/Table"
+
+async function main(assetTypes) {
+  // [START asset_quickstart_list_assets]
+  const util = require('util');
+  const {v1p5beta1} = require('@google-cloud/asset');
+  const client = new v1p5beta1.AssetServiceClient();
+
+  const projectId = await client.getProjectId();
+  const projectResource = `projects/${projectId}`;
+  // TODO(developer): Choose types of assets to list, such as 'storage.googleapis.com/Bucket':
+  //   const assetTypesList = ['storage.googleapis.com/Bucket', 'bigquery.googleapis.com/Table'];
+  // Or simply use empty list to list all types of assets:
```

Comment:
what we tend to do in comments here, is leave the expected variable commented out, such that someone could simply remove the `//` and have a functional sample:

https://github.com/googleapis/nodejs-secret-manager/blob/master/samples/getSecretVersion.js#L22

Reply:
Got it. I changed the comments to use "const assetTypes = ..." directly, in order to align with the params used below.